src/printer.js

```diff
@@ -2724,6 +2656,7 @@ function printStatementSequence(path, options, print) {
 
     // Just in case the AST has been modified to contain falsy
     // "statements," it's safer simply to skip them.
+    /* istanbul ignore if */
     if (!stmt) {
       return;
     }
```

Comment:
It's probably for codemod purposes but we don't deal with them right now. Should we remove this for now?